data/class/pages/admin/products/LC_Page_Admin_Products_Product.php

```diff
@@ -1378,12 +1380,14 @@ public function hasSubProductData($arrSubProductData)
         $has_subproduct_data = false;
 
         for ($i = 1; $i <= PRODUCTSUB_MAX; $i++) {
-            if (SC_Utils_Ex::isBlank($arrSubProductData['sub_title'.$i]) == false
+            if (
+                SC_Utils_Ex::isBlank($arrSubProductData['sub_title'.$i]) == false
                 || SC_Utils_Ex::isBlank($arrSubProductData['sub_comment'.$i]) == false
                 || SC_Utils_Ex::isBlank($arrSubProductData['sub_image'.$i]) == false
                 || SC_Utils_Ex::isBlank($arrSubProductData['sub_large_image'.$i]) == false
                 || SC_Utils_Ex::isBlank($arrSubProductData['temp_sub_image'.$i]) == false
                 || SC_Utils_Ex::isBlank($arrSubProductData['temp_sub_large_image'.$i]) == false
+                || $arrSubProductData['image_key'] == 'sub_image'.$i
```

Comment:
```suggestion
                || $arrSubProductData['image_key'] == 'sub_image'.$i
                || $arrSubProductData['image_key'] == 'sub_large_image'.$i
```

最初にサブ拡大画像にアップロードすると閉じてしまったので、`'sub_large_image'.$i` も追加でお願いします